src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.csproj

```diff
@@ -3,7 +3,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <IsPackable>false</IsPackable>
     <!-- Avoid ID conflicts with the package project. -->
-    <PackageId>*$(MSBuildProjectFullPath)*</PackageId>
+    <PackageId>Real.Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers</PackageId>
```

Comment:
>Real [](http://example.com/codeflow?start=15&length=4)

Real?  Would package be generated with this ID? what happens with old package id?

Reply:
No package will be generated with this ID. This project doesn't get package individually. There is a Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Package.csproj that handles the packaging of this project (and a few other projects).

If this project did get packaged, then the old PackageId would have evaluated to something like `C:\...source\upgrade-assistant\...\Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.csproj` which, obviously, would never work as an actual package name.